### PR TITLE
refactor: use map for environment instead of pipes

### PR DESCRIPTION
**Pull request by robot9706**
Agents currently use pipe separated strings for environment variables. Refactored to maps.

### CODE REVIEW

File: web/crux/.prettierrc [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "arrowParens": "avoid",
   "bracketSpacing": true,
-  "endOfLine": "lf",
+  "endOfLine": "crlf",
```
Comment (m8vago) [inline web/crux/.prettierrc:4]: Why?

Comment (robot9706) [reply web/crux/.prettierrc:4]: Oops, sorry :)

**Review by m8vago — APPROVED** [review]
LGTM. @nandor-magyar ?

**Review by nandor-magyar — APPROVED** [review]
Looking good.

**Pull request merged**